Editor/Graph/Renderer/DefaultGraphRenderer.cs

```diff
@@ -73,7 +73,7 @@ public void Draw(IGraphLayout graphLayout, Rect totalDrawingArea, GraphSettings
 
             PrepareLegend(graphLayout.vertices);
 
-            if (graphSettings.showLegend || graphSettings.showInspector)
+            if (graphSettings.showInspector)
```

Comment:
This still works for my needs, but unfortunately leaving out .showLegend means that if you toggle only the legend, then it won't show, so essentially you only see the legend when the inspector is shown. If that is desired, then I'm fine with it, but you may want to modify the toggles, so that inspector gets enabled when only the legend is enabled. I was considering the inspector as separate from the legend, but if you consider the legend to be contained within the inspector, then I guess it works.